app/furniture/marketplace/order_notification_method_component.rb

```diff
@@ -0,0 +1,36 @@
+class Marketplace
+  class OrderNotificationMethodComponent < ApplicationComponent
+    attr_accessor :order_notification_method
+    delegate :contact_location, to: :order_notification_method
+
+    def initialize(order_notification_method:, **kwargs)
+      super(**kwargs)
+
+      self.order_notification_method = order_notification_method
+    end
+
+    def edit_button
+      super(title: t("marketplace.order_notification_methods.edit.link_to", contact_location: contact_location),
+            href: order_notification_method.location(:edit))
+    end
```

Comment:
It would be nice if `#edit_button`, `#edit_button?`, `#destroy_button`, and `#destroy_button?` could be part of a `RecordComponent` or something so we don't have to define them every time.